packages/eslint-plugin-testing-library-react/src/rules/prefer-expect-query-by.ts

```diff
@@ -58,7 +58,7 @@ function mapNodesForWrongGetByQuery(node: TSESTree.CallExpression) {
 function hasExpectWithWrongGetByQuery(node: TSESTree.CallExpression) {
   if (
     node.callee &&
-    isIdentifier(node.callee) &&
+    node.callee.type === AST_NODE_TYPES.Identifier &&
```

Comment:
Using `isIdentifier` here makes the compiler confused

<img width="660" alt="image" src="https://user-images.githubusercontent.com/1110551/64103163-dd575f00-cd71-11e9-8055-9c643043d3fe.png">